sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/AvroGenericRecordToStorageApiProto.java

```diff
@@ -69,11 +71,15 @@ public class AvroGenericRecordToStorageApiProto {
   // A map of supported logical types to the protobuf field type.
   static final Map<String, TableFieldSchema.Type> LOGICAL_TYPES =
       ImmutableMap.<String, TableFieldSchema.Type>builder()
-          .put(LogicalTypes.date().getName(), TableFieldSchema.Type.DATE)
-          .put(LogicalTypes.decimal(1).getName(), TableFieldSchema.Type.BIGNUMERIC)
-          .put(LogicalTypes.timestampMicros().getName(), TableFieldSchema.Type.TIMESTAMP)
-          .put(LogicalTypes.timestampMillis().getName(), TableFieldSchema.Type.TIMESTAMP)
-          .put(LogicalTypes.uuid().getName(), TableFieldSchema.Type.STRING)
+          .put("date", TableFieldSchema.Type.DATE)
```

Comment:
what's the reason to change them to hard coded names here? I understand they should be equivalent? Or keep the getName() while note the resolved name as comments?

Reply:
both are fine, if choose to go with resolved names, good to add a comment these come from corresponding avro LogicalTypes implementations' getName()

Reply:
This code has been refactored to accomodate conversion from avro `decimal` type to BQ `NUMERIC/BIGNUMERIC`